src/thread_manager.ts

```diff
@@ -34,17 +38,18 @@ export class ThreadManager<SCG extends ExtendableGenerics = DefaultGenerics> {
     this.client = client;
     this.state = new StateStore<ThreadManagerState<SCG>>({
       active: false,
-      // existing threads which have gotten recent activity during the time the manager was inactive
-      existingReorderedThreadIds: [],
+      isThreadOrderStale: false,
       threads: [],
       threadIdIndexMap: {},
```

Comment:
```suggestion
```
I feel like this can be dropped if it's no longer being used anywhere.

Reply:
Also - what was the reason for the drop? I feel like with large influx of replies across multiple loaded threads could make a dent in performance with all the slow-ish `findIndex` ops - no (talking about reply handler)?

Reply:
Restored the lookup table, but now it's updated together with threads: https://github.com/GetStream/stream-chat-js/pull/1348/commits/ab0db2fee91ce332598cb2619562489da55b5bf9.